package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mask-network",
-  "packageManager": "pnpm@7.1.1",
```

Comment:
Looks like you didn't merge the develop branch correctly. You're reverting many changes.